### ISSUE TITLE
"An illegal reflective access operation has occurred" warning when trying to use Detekt with Java >=9

### ISSUE THREAD

**Issue by remal** [arturbosch/detekt#1693]
I'm trying to use Detekt with Gradle and Java 12.

And I'm getting this warnings:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jetbrains.kotlin.com.intellij.openapi.util.JDOMUtil$2 (file:/C:/Users/semen_levin/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-compiler-embeddable/1.3.31/965ef6d9abe1555cc5d0a34dd24ce93b49123e7c/kotlin-compiler-embeddable-1.3.31.jar) to constructor com.sun.xml.internal.stream.XMLInputFactoryImpl()
WARNING: Please consider reporting this to the maintainers of org.jetbrains.kotlin.com.intellij.openapi.util.JDOMUtil$2
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

Please fix or hide this warnings.

## Your Environment
* Version of detekt used: Gradle plugin 1.0.0-RC15
* Version of Gradle used (if applicable): 5.4.1
* Operating System and version: Windows 10
* Java version: 12

**Comment by schalkms** [comment]
Thanks for reporting this warning.
@arturbosch can we do anything about this besides reporting this to the repo of _.com.intellij_?
I mean this a minor problem.

**Comment by arturbosch** [comment]
It's a know issue on their side: https://youtrack.jetbrains.net/issue/KT-30387

**Comment by 3flex** [comment]
Suggest closing this one since it affects code that detekt has no control over, so there's nothing that can be done here...
